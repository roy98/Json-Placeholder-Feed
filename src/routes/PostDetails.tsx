import React from 'react'
import MainLayout from '../components/layout/mainLayout'
import SinglePost from '../components/posts/SinglePost'

function PostDetail() {

  return (
    <>
        <MainLayout>
            <SinglePost />
        </MainLayout>
    </>
  )
}

export default PostDetail