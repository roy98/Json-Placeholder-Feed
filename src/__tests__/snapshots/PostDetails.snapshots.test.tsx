import React from 'react'
import renderer from "react-test-renderer"
import { BrowserRouter } from "react-router-dom"
import PostDetails from "../../routes/PostDetails"

test("renders details page of a post", () => {
  const tree = renderer.create(
    <BrowserRouter>
        <PostDetails />
    </BrowserRouter>
  ).toJSON();
  
  expect(tree).toMatchSnapshot();
});