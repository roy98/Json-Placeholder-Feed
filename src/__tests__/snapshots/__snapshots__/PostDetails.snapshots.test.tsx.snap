// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders details page of a post 1`] = `
<div
  id="container"
>
  <header>
    <nav>
      <h2>
        <a
          href="/"
          onClick={[Function]}
        >
          JSON Placeholder Feed
        </a>
      </h2>
    </nav>
  </header>
  <main>
    <div
      className="main-content"
    >
      <div>
        <div
          className="single-post-container"
        >
          <div
            className="single-post-card"
          >
            <h4
              className="single-post-card-title"
            />
            <p
              className="single-post-card-body"
            />
          </div>
        </div>
        <div
          className="separator-vertical"
        />
        <div
          className="comments-container"
        >
          <h3>
            Comments
          </h3>
          <div
            className="comments-grid"
          >
            <button
              className="btn btn-disabled"
              disabled={true}
              onClick={[Function]}
            >
              Load more
            </button>
          </div>
        </div>
      </div>
    </div>
  </main>
  <footer>
    <strong>
      JSON Placeholder Feed
    </strong>
    <p>
      Â©Copyright 
      2023
    </p>
    <code>
      Test Powered by Digit-AD
    </code>
  </footer>
</div>
`;
